app/src/main/res/layout/context_menu.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
nit: the file header should stay on the next line. It's unnecessary churn and, imo, it makes more sense on the next line. Here and in the next file.

Reply:
Oh sorry that's the autoformatter.